src/hb-unicode-emoji-table.hh

```diff
@@ -7,15 +7,15 @@
  * on file with this header:
  *
  * # emoji-data.txt
- * # Date: 2019-01-15, 12:10:05 GMT
- * # © 2019 Unicode®, Inc.
+ * # Date: 2020-01-28, 20:52:38 GMT
+ * # © 2020 Unicode®, Inc.
  * # Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
- * # For terms of use, see https://www.unicode.org/terms_of_use.html
+ * # For terms of use, see http://www.unicode.org/terms_of_use.html
  * #
  * # Emoji Data for UTS #51
- * # Version: 12.0
+ * # Version: 13.0
  * #
- * # For documentation and usage, see https://www.unicode.org/reports/tr51
+ * # For documentation and usage, see http://www.unicode.org/reports/tr51
```

Comment:
Oh, where these http links, instead https, are coming from? Just nice to have https links back :)

Reply:
This is copied from emoji-data.txt.